CMakeLists.txt

```
@@ -368,7 +368,7 @@ if(ACTS_BUILD_PLUGIN_EDM4HEP)
   find_package(podio ${_acts_podio_version} REQUIRED CONFIG)
   find_package(EDM4HEP ${_acts_edm4hep_version} REQUIRED CONFIG)
 endif()
-if(ACTS_BUILD_EXAMPLES_GEANT4)
+if(ACTS_BUILD_EXAMPLES_GEANT4 OR ACTS_BUILD_PLUGIN_GEANT4)
```

```suggestion
if(ACTS_BUILD_PLUGIN_GEANT4)
```

@H4m5t3r can you change this?